regression/contracts/quantifiers-loop-01/test.desc

```diff
@@ -1,11 +1,13 @@
 CORE
 main.c
---apply-loop-contracts
+--apply-loop-contracts _ --trace
```

Comment:
Why do we need to get the trace here? I don't see any regex specific to it.

Reply:
fixed